oci-umount.conf

```diff
@@ -6,3 +6,5 @@
 /var/lib/docker-latest/overlay
 /var/lib/docker-latest/devicemapper
 /var/lib/docker-latest/containers
+/var/lib/container/storage/overlay
+/var/lib/container/storage/overlay-containers
```

Comment:
Has somebody confirmed if this if `overlay-containers` is a real directory created by somebody.